src/userRepo.js

```diff
@@ -26,23 +26,16 @@ class UserRepo {
     return this.users.find((user) => user.id === id);
   }
 
-  // could dry up the three functions below. pass in parameter (fullList). take in 2 arguments (water, ID) fullList and ID
-  // low hanging fruit
-
   filterHydrationData(id) {
-    return this.hydrationData.filter(
-      (hydrationItem) => hydrationItem.userID === id
-    );
+    return this.hydrationData.filter((hydrationItem) => hydrationItem.userID === id);
```

Comment:
nit: Is styling here intentional? If a fat arrow function accepts 1 arg, the surrounding parenthesis can be omitted.

Reply:
Extra () removed throughout file